esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -326,15 +333,24 @@ struct WUComponentLogOptions
         logFetchOptions.setStartFrom(logFilterReq.getLineStartFrom());
 
         if (logFilterReq.getComponentsFilter().length() > 0)
-            compoundOwnedFilter(logFetchFilter, getOredComponentsLogFilter(logFilterReq.getComponentsFilter()), LOGACCESS_FILTER_and);
+        {
+            Owned<ILogAccessFilter> filterClause = getOredComponentsLogFilter(logFilterReq.getComponentsFilter());
```

Comment:
trivial: there's a mix of styles here assigning to Owned<ILogAccessFilter>, here by assignment,
in other places via the ctor (e.g. line 274). Either fine, no real benefit of either, assignment is more typical.